src/ExplainProvenanceImpl.h

```diff
@@ -780,7 +780,6 @@ class ExplainProvenanceImpl : public ExplainProvenance {
                 return std::make_tuple(ruleNum, levelNum);
             }
         }
-
```

Comment:
The presence or absence of this line is fine, but either way it's not worth adding the extra file to the commit. Please be a little more selective with the files you commit.